egui/src/demos/demo_window.rs

```diff
@@ -68,9 +70,7 @@ impl DemoWindow {
         CollapsingHeader::new("Scroll area")
             .default_open(false)
             .show(ui, |ui| {
-                ScrollArea::from_max_height(200.0).show(ui, |ui| {
-                    ui.label(LOREM_IPSUM_LONG);
-                });
```

Comment:
I think it would be good to keep a simple scroll demo to demonstrate how to create a simple `ScrollArea`.